docs/apis/subsystems/task/adhoc.md

```diff
@@ -156,3 +156,40 @@ The `reschedule_or_queue_adhoc_task()` function will locate any existing task ma
 ```php
 \core\task\manager::reschedule_or_queue_adhoc_task($task);
 ```
+
+#### Set the retry limit for a task
+
+<Since version="4.4" issueNumber="MDL-79128" />
+
+Since Moodle 4.4, the maximum number of retries for a task is **12** by default.
```

Comment:
Again:

The number of retries for a failing task can be controlled using the `set_attempts_avaiable` method on the task before it is queued. If not specified a task will be allowed 12 retries.